src/elements/content-preview/ContentPreview.js

```diff
@@ -362,11 +362,16 @@ class ContentPreview extends React.PureComponent<Props, State> {
      * @return {void}
      */
     componentDidUpdate(prevProps: Props, prevState: State): void {
-        const { token } = this.props;
-        const { token: prevToken } = prevProps;
+        const { token, previewExperiences } = this.props;
```

Comment:
Minor, but we generally try to alphabetize our props, object properties, imports, etc. when possible.